packages/knit-gateway/src/headers.test.ts

```diff
@@ -37,7 +42,7 @@ describe("well known headers", () => {
 });
 
 describe("well known prefix", () => {
-  const prefixes = [":", "accept-", "Connect-", "Content-", "IF-"];
+  const prefixes = [":", "accept-", "Connect-", "Content-", "IF-", "Grpc-"];
```

Comment:
nit: it would be nice if these used consistent case.
```suggestion
  const prefixes = [":", "Accept-", "Connect-", "Content-", "If-", "Grpc-"];
```